content/kapacitor/v1.5/working/_index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Working with Kapacitor
+title: Work with Kapacitor
 
```

Comment:
Needs a description. What about:

Use Kapacitor to collect, monitor and alert on time series data.